AssetManagementTestProject/Asserter/AssetListAsserter.cs

```diff
@@ -0,0 +1,34 @@
+﻿//using AssetManagementTestProject.DAO;
+//using NUnit.Framework;
+
+//// ------------------------------- IN CASE FLUENT ASSERTION FAILS -------------------------------
+//namespace AssetManagementTestProject.Asserter;
+
+//public class AssetListAsserter
+//{
+
+//    public static void AssertViewAssetLists(IEnumerable<AssetDAO> expectedUsers, 
+//        IEnumerable<AssetDAO> actualUsers)
+//    {
+//        // use when asserting two user list in ViewUser
+//        Asserter.AssertListEquals((e, a) => AssertViewAsset(e, a), expectedUsers, actualUsers);
+//    }
+//    public static void AssertViewAsset(AssetDAO expected, AssetDAO actual)
+//    {
```

Comment:
Removed unused classes

Reply:
I commented it out in case fluent assertions fails and I am too lazy to write it all again 😅

Reply:
The fluent assertions should be worked at the stable version. So in this case, your problem is that:
- How to write the HTML log for these assertions?
Just think about the hooks to implement this or remove all unused code. Then you have to look at this in the implementation tickets

Reply:
Working on it atm. thanks for the suggestion! Will resolve this when I figure it out